agave_app/loadDialog.cpp

```diff
@@ -172,6 +175,8 @@ LoadDialog::LoadDialog(std::string path, const std::vector<MultiscaleDims>& dims
   layout->addItem(new QSpacerItem(0, spacing, QSizePolicy::Expanding, QSizePolicy::Expanding));
   layout->addRow(mMemoryEstimateLabel);
   layout->addItem(new QSpacerItem(0, spacing, QSizePolicy::Expanding, QSizePolicy::Expanding));
+  layout->addRow("Keep current AGAVE settings", m_keepSettingsCheckbox);
+  layout->addItem(new QSpacerItem(0, spacing, QSizePolicy::Expanding, QSizePolicy::Expanding));
```

Comment:
Nit: AGAVE settings to me suggests the configuration/settings for the entire AGAVE application (I don't know if we actually have saved preferences, now that I think about it), rather than just the current viewer settings. Would "viewer settings" or "volume settings" be more clear?